scripts/plugins/firewall-port

```diff
@@ -19,6 +19,11 @@ PROTOCOL="${3:-tcp}"
 CHAIN="xapi-INPUT"
 RULE="-p $PROTOCOL -m conntrack --ctstate NEW -m $PROTOCOL --dport $PORT -j ACCEPT"
 
+if [ "$PORT" -eq "80" ]
```

Comment:
I think it would be clearer to `CHAIN="xapi-INPUT"` in and `else` branch. Another idea is to use `case` over `$PORT` as this would make adding cases easier.

Reply:
Would this new chain be used only for port 80 or on others?

Reply:
The current logic could be encoded like this:

```
case "$PORT" in
  80)
    CHAIN="RH-Firewall-1-INPUT"
    ;;
  *)
    CHAIN="xapi-INPUT"
    ;;
esac
```

And my argument is, that this is anticipating future cases. A simpler solution is to use `if then else fi`, which I would prefer over having a default which is over written.